salt/journal/config/srv-journal-local-demo-app-config-parameters.yml

```diff
@@ -36,3 +36,5 @@ parameters:
     hypothesis_authority: some-authority
     hypothesis_client_id: some-id
     hypothesis_client_secret: ThisIsASecret
+
+    feature_can_always_use_hypothesis: true
```

Comment:
It already uses `?open-sesame`, so this value doesn't matter.

Reply:
(as long as the bootstrap runs)